internal/consensus/state_test.go

```diff
@@ -1139,12 +1503,13 @@ func TestStateLockPOLSafety1(t *testing.T) {
 
 // What we want:
 // dont see P0, lock on P1 at R1, dont unlock using P0 at R2
-func TestStateLockPOLSafety2(t *testing.T) {
+func TestStateLock_POLSafety2(t *testing.T) {
```

Comment:
does it make sense to use subtests for these related tests?

Reply:
The functionality being tested is conceptually related, but the conditions needed to actually perform the different tests are so different that the subtests would basically be as long as the two tests pasted next to each other, so I'm not sure that that's much of an improvement.